frontend/pages/beta/index.tsx

```diff
@@ -0,0 +1,161 @@
+import SearchIcon from '@mui/icons-material/Search'
```

Comment:
Similar comment to before about maybe combining these into one import.  And the below imports being made absolute by using '@' instead of '../..'

Reply:
@ Isn't working for me!